tests/functional/test_app_mainmenu_settings_cfg.py

```diff
@@ -17,22 +17,23 @@ def test_app_mainmenu_settings_cfg(backend, navigator, test_name):
     assert data_allowed is False
     assert version == (MAJOR, MINOR, PATCH)
 
-    # Navigate in the main menu and the setting menu
-    # Change the "data_allowed parameter" value
-    instructions = [
-        NavIns(NavInsID.RIGHT_CLICK),
-        NavIns(NavInsID.RIGHT_CLICK),
-        NavIns(NavInsID.RIGHT_CLICK),
-        NavIns(NavInsID.LEFT_CLICK),
-        NavIns(NavInsID.BOTH_CLICK),
-        NavIns(NavInsID.BOTH_CLICK),
-        NavIns(NavInsID.RIGHT_CLICK),
-        NavIns(NavInsID.BOTH_CLICK)
-    ]
-    navigator.navigate_and_compare(ROOT_SCREENSHOT_PATH, test_name, instructions,
-                                   screen_change_before_first_instruction=False)
+    if type(backend).__name__ == "SpeculosBackend":
```

Comment:
Could you use `if isinstance(client, SpeculosBackend):` to be consistent with what is done in `test_pubkey_cmd.py`?
And could you add a comment which explains why this is only done over speculos?
Thanks!